kura/org.eclipse.kura.ai.triton.server/OSGI-INF/metatype/org.eclipse.kura.ai.triton.server.TritonServerContainerService.xml

```diff
@@ -73,6 +73,15 @@
             description="A comma separated list of inference model names that the server will load.">
         </AD>
 
+        <AD id="local.backends.path"
+            name="Local backends path"
+            type="String"
+            cardinality="0"
+            required="false"
+            default=""
+            description="Specify the filesystem path where the backends are stored.">
```

Comment:
I think we should point out that this *overrides the backend stored within the container* with the backend found on the host filesystem. This behaviour is quite nuanced. Could be something like:

```suggestion
            description="Specify the host filesystem path where the backends are stored. If left blank, the backends provided by the Triton container will be used.">
```